solver/vertex.go

```diff
@@ -17,6 +18,7 @@ type Vertex interface {
 	// Sys returns an internal value that is used to execute the vertex. Usually
 	// this is capured by the operation resolver method during solve.
 	Sys() interface{}
+	SysMetadata() pb.OpMetadata
```

Comment:
just `Metadata()`, and maybe return `solver.Metadata` interface? Atm the solver should be usable without llb definition.

Reply:
Yes, updated PR.

However, Solver still has depend on the `pb` package:
```go
func (s *Solver) Solve(ctx context.Context, id string, f frontend.Frontend, def *pb.Definition, exp exporter.ExporterInstance, frontendOpt map[string]string) error
```

Reply:
Ok, yes I think I broke it when I moved the LLB-to-vertex conversion to the `job.load`. Need to think about it later if there is a way around that.